analyzer/src/funTest/assets/projects/external/dart-http-expected-output.yml

```diff
@@ -1286,7 +1286,7 @@ packages:
   - "Dart Team"
```

Comment:
Commit message: The body is not a full sentence; I think the first word is wrong.